administrationService/src/main/scala/pl/edu/agh/iosr/cookieHeaven/administration/db/Offer.scala

```diff
@@ -0,0 +1,9 @@
+package pl.edu.agh.iosr.cookieHeaven.administration.db
+
+import org.springframework.data.mongodb.core.mapping.Document
+
+/**
+  * @author Wojciech Pachuta.
+  */
+@Document
+case class Offer(id: String, name: String, price: Double)
```

Comment:
czemu to zduplikowales? modul core jest dodany wszedzie do zaleznosci, wszystko co tam jest powinno byc widocze w order i admin serwisach

Reply:
@Ugon  jest widoczne, ale sie wypieprza przy startowaniu spring-boota z błędem:
```
2017-12-03 19:44:29.560  WARN 17508 --- [           main] ationConfigEmbeddedWebApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'offerService' defined in file [/home/kuba/workspace/studies/IOSR/cookie-heaven/administrationService/target/classes/pl/edu/agh/iosr/cookieHeaven/administration/service/OfferService.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'offerRepository': Invocation of init method failed; nested exception is org.springframework.data.mapping.PropertyReferenceException: No property id found for type Offer!
2017-12-03 19:44:29.565  INFO 17508 --- [           main] o.apache.catalina.core.StandardService   : Stopping service Tomcat
2017-12-03 19:44:29.579  INFO 17508 --- [           main] utoConfigurationReportLoggingInitializer : 

Error starting ApplicationContext. To display the auto-configuration report re-run your application with 'debug' enabled.
2017-12-03 19:44:29.588 ERROR 17508 --- [           main] o.s.boot.SpringApplication               : Application startup failed

org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'offerService' defined in file [/home/kuba/workspace/studies/IOSR/cookie-heaven/administrationService/target/classes/pl/edu/agh/iosr/cookieHeaven/administration/service/OfferService.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'offerRepository': Invocation of init method failed; nested exception is org.springframework.data.mapping.PropertyReferenceException: No property id found for type Offer!
```
i nie wiem jak sobie z tym poradzic, masz może pomysł?